config/packages/_sylius.yaml

```
@@ -14,6 +14,8 @@ sylius_core:
         bc_layer:
             enabled: true
             routes:
+                admin_administrators:
```

```suggestion
                admin_admin_user:
```
WDYT? It looks a bit weird, but it's the name used in configs and is in line with the routes themselves, `sylius_admin_admin_user_create` etc.

---

Yes, I was mitigated. The Behat suite is named ui_managing_administrators and the Behat tag too.